lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -0,0 +1,302 @@
+;(function (define, undefined) {
+    'use strict';
+    define([
+        'gettext', 'jquery', 'underscore', 'backbone', 'js/vendor/backbone-super',
+    ], function (gettext, $, _, Backbone) {
+
+        var messageRevertDelay = 4000;
+        var AccountSettingsFieldViews = {};
+
+        AccountSettingsFieldViews.FieldView = Backbone.View.extend({
```

Comment:
@andy-armstrong I have updated the templates to not use `account-settings` in class names and moved them to `templates/fields`.  Is moving this file to a common directory and changing `AccountSettingsFieldViews` to just `FieldViews` sufficient for now or are you thinking something else?